partner_firstname/tests/test_delete.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2015 Grupo ESOC
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo.tests.common import TransactionCase
+
+from .base import MailInstalled
+
+
+class CompanyCase(TransactionCase):
+    model = "res.partner"
+    context = {"default_is_company": True}
+
+    def test_computing_after_unlink(self):
+        """Test what happens if recomputed after unlinking.
+
+        This test might seem useless, but really this happens when module
+        ``partner_relations`` is installed.
+
+        See https://github.com/OCA/partner-contact/issues/154.
+        """
+        data = {"name": "Söme name"}
+        # pylint: disable=W8121
+        record = self.env[self.model].with_context(self.context).create(data)
```

Comment:
with_context(**self.context) to avoid the pylint check issue more than disabling the check

Reply:
Will try to make it work

Reply:
That's not a reason to keep. On every migration we should improve things if possible